DOC: Proofreading ethernet.rst (#1740)
vyos/vyos-documentation: docs/_include/interface-vlan-8021ad.txt

**Overview**

IEEE 802.1ad_, commonly known as QinQ, is an Ethernet standard first published 
as an amendment to 802.1q_ in 2005, then officially merged into the base 
standard in 2011.

Unlike the original 802.1q_, which allows a single VLAN header per Ethernet 

> frame, QinQ allows multiple VLAN headers per Ethernet frame.

Reviewer: ```suggestion
Unlike the original 802.1q_, which allows a single VLAN header per Ethernet 
frame, QinQ allows two VLAN headers per Ethernet frame, for the inner and the outer VLAN tags.
Most often the inner VLAN tag comes from a customer while the outer tag is used by the service
provider to differentiate between traffic of different customers.
```

In the evolution of Ethernet, QinQ serves as a bridge: it extends 802.1q_ and 
forms the basis for later Metro Ethernet technologies. Its support for multiple 
VLAN headers is essential for Metro Ethernet, allowing service providers to 

> encapsulate and segregate customer traffic on shared infrastructure.

Reviewer: ```suggestion
```

This is factually incorrect. The concept of "Metro Ethernet" predates QinQ. In https://web.archive.org/web/20140327090207/http://metroethernetforum.org/Assets/Technical_Specifications/PDF/MEF4.pdf (published in 2004), you will find:

> The MEF architecture model is based
on the principle of layer network decomposition, where each layer network is constructed from a
particular set of networking technologies (e.g., Ethernet, SONET/SDH, MPLS, etc.)

Note that the definitions are intentionally abstracted away from any particular way to separate Ethernet traffic flow: 

>The Ethernet Virtual Connection (EVC) is the architecture construct that supports the association
of UNI reference points for the purpose of delivering an Ethernet flow between subscriber sites
across the MEN.

Development of QinQ [started in 2002](https://www.ieee802.org/1/pages/802.1ad.html) so anything marketed as Metro Ethernet and based on standards would have to use something else.

In practice, all such networks are built on top of MPLS cores now. I cannot find the original technical specifications but my suspicious is that anything built on top of QinQ alone would _ever_ satisfy the requirements of the Metro Ethernet Forum specs that define a lot of things about EVCs and their attributes related to frame delivery policies.

It's best not to mention Metro Ethernet here at all.

Author: Removed.

**Frame structure and ethertypes**

The IEEE 802.1ad_ (QinQ) frame includes two nested 802.1q_ VLAN headers, 

> commonly referred to as VLAN tags, or simply tags:

Reviewer: ```suggestion
The IEEE 802.1ad_ (QinQ) frame includes two VLAN tags:
```

They are not nested, just placed next to each other in the header.

<img width="656" height="291" alt="Image" src="https://github.com/user-attachments/assets/15436b44-c5c1-41c5-b9bb-8a0b3d93b107" />

Author: Corrected.

* **The outer service tag (S-TAG):** Located closest to the Ethernet header, 
  the S-TAG is typically added by the provider's edge bridge to route traffic 

>   through the Metro Ethernet network. It uses the Ethertype 0x88a8.

Reviewer: ```suggestion
* **The outer service tag (S-TAG):** The S-TAG is typically added by the provider.
It uses the Ethertype 0x88a8 by default. 

```

I'd say it _is_ a part of the header. In any case, that information is irrelevant for the user — people who are curious about the frame header format can look it up.

Regarding the protocol, there is an option to set it to 802.1q (`set interfaces ethernet ethX vif-s <N> protocol <802.1ad|802.1q>`. It's not used often anymore now that 802.1ad is a standard but some legacy, pre-standard implementations may still exist in the wild.

Author: Removed.

* **The inner customer tag (C-TAG):** Encapsulated within the S-TAG and located 
  closest to the payload, the C-TAG is generated by the customer's equipment and 

>   remains unchanged during transit. It uses the Ethertype 0x8100.

Reviewer: ```suggestion
* **The inner customer tag (C-TAG):** The C-TAG is generated by the customer's equipment and 
  remains unchanged during transit. It uses the Ethertype 0x8100.
```

Author: Corrected.

**Implementation in VyOS**

> In VyOS, these tag types are represented as:

Reviewer: ```suggestion
In VyOS, these tag types are associated with the following CLI options:
```
